site/src/components/UserAutocomplete/UserAutocomplete.tsx

```diff
@@ -111,7 +112,7 @@ export const useStyles = makeStyles((theme) => {
       },
 
       "& input": {
-        fontSize: 14,
```

Comment:
I think instead of doing the styles here we could:
- Remove the "customized" styles completely and add a className props. Change styles on base components usually break things in other places where it is used.

Reply:
Sure, I can add that prop! I adjusted the `fontSize` because I noticed our other inputs used size 16. Should I switch it back? @BrunoQuaresma